wilds/datasets/gwhd_dataset.py

```diff
@@ -0,0 +1,143 @@
+import numpy as np
+import pandas as pd
+import torch
+from pathlib import Path
+from PIL import Image
+from wilds.datasets.wilds_dataset import WILDSDataset
+from wilds.common.grouper import CombinatorialGrouper
+from wilds.common.metrics.all_metrics import DetectionAccuracy
+
+
+def _collate_fn(batch):
+    """
+    Stack x (batch[0]) and metadata (batch[2]), but not y.
+    """
+    batch = list(zip(*batch))
+    batch[0] = torch.stack(batch[0])
+    batch[2] = torch.stack(batch[2])
+    return tuple(batch)
+
+class GWHDDataset(WILDSDataset):
+    """
+    The GWHD-wilds wheat head localization dataset.
+    This is a modified version of the original Global Wheat Head Dataset.
+    This dataset is not part of the official WILDS benchmark.
+    We provide it for convenience and to reproduce observations discussed in the WILDS paper.
+    Supported `split_scheme`:
+        'official' for WILDS related tasks.
+        To reproduce the baseline, several splits are needed:
+        - to train a model on train domains and test against a all test split: 'train_in-dist'
+        - "benchmark_biased" ; "benchmark_in-dist"
+    Input (x):
+        1024x1024 RGB images of wheat field canopy between flowering and ripening.
+    Output (y):
+        y is a nx4-dimensional vector where each line represents a box coordinate (x_min,y_min,x_max,y_max)

```

Comment:
Just to double check, is this coordinate format still correct? (I know we had to change it for DETR)